PL2/PL2-parser/src/main/java/nl/tudelft/pl2016gr2/parser/controller/FullGfaReader.java

```diff
@@ -115,6 +122,35 @@ private void handleL(Scanner sc) {
     ch.addInlink(par.getId());
     sc.nextLine();
   }
+  
+  /**
+   * This method extracts the single genomes from the ORI tag.
+   * @param line the full ORI tag
+   * @return an arrayList of the names of the genomes.
+   */
+  private ArrayList<String> extractGenomes(String line) {
+    String[] words = line.split(":");
+    if (words[0].endsWith("ORI")) {
+      String[] gens = words[2].split(";");
+      ArrayList<String> gensAr = new ArrayList<>();
+      gensAr.addAll(Arrays.asList(gens));
+      return gensAr;
```

Comment:
You can instantiate an arraylist with a collection as argument:
return new ArrayList<>(Arrays.asList(gens));